sci-libs/gdal/gdal1-1.11.5.recipe

```diff
@@ -116,26 +124,70 @@ BUILD_PREREQUIRES="
 	cmd:pkg_config$secondaryArchSuffix
 	"
 
-defineDebugInfoPackage gdal1$secondaryArchSuffix \
-	$libDir/libgdal.so.$libVersion
+debugList=("$libDir"/libgdal.so.$libVersion)
+if [ -z "$commandSuffix" ]; then
+debugList+=(
+	"$commandBinDir"/gdal_contour \
+	"$commandBinDir"/gdal_grid \
+	"$commandBinDir"/gdal_rasterize \
+	"$commandBinDir"/gdal_translate \
+	"$commandBinDir"/gdaladdo \
+	"$commandBinDir"/gdalbuildvrt \
+	"$commandBinDir"/gdaldem \
+	"$commandBinDir"/gdalenhance \
+	"$commandBinDir"/gdalinfo \
+	"$commandBinDir"/gdallocationinfo \
+	"$commandBinDir"/gdalmanage \
+	"$commandBinDir"/gdalserver \
+	"$commandBinDir"/gdalsrsinfo \
+	"$commandBinDir"/gdaltindex \
+	"$commandBinDir"/gdaltransform \
+	"$commandBinDir"/gdalwarp \
+	"$commandBinDir"/nearblack \
+	"$commandBinDir"/ogr2ogr \
+	"$commandBinDir"/ogrinfo \
+	"$commandBinDir"/ogrlineref \
+	"$commandBinDir"/ogrtindex \
+	"$commandBinDir"/testepsg
+	)
+fi
+defineDebugInfoPackage gdal1$secondaryArchSuffix "${debugList[@]}"
 
 BUILD()
 {
 	expatDir=$portPackageLinksDir/lib~libexpat
 	autoreconf -fi
-	runConfigure ./configure --with-expat=$expatDir
+	runConfigure --omit-dirs "binDir dataDir" ./configure \
+		--bindir "$commandBinDir" \
+		--datadir="$dataDir"/gdal1 \
+		--with-expat=$expatDir
 	make $jobArgs
 }
 
 INSTALL()
 {
 	make install
 
+	if [ -n "$secondaryArchSuffix" ]; then
+		mkdir -p "$binDir"
+		mv "$commandBinDir"/gdal-config "$binDir"
+	fi
+
 	prepareInstalledDevelLib libgdal
 	fixPkgconfig
 
 	# devel package
 	packageEntries devel \
-		$binDir/gdal-config \
-		$developDir
+		"$binDir"/gdal-config \
+		"$developDir"
+
+
```

Comment:
> @fbrosson I'm still left with an empty "x86" folder in $binDir ...

Good catch. Here's a possible fix. It should work (but I did not test it yet.)
```diff
 	# devel package
 	packageEntries devel \
 		"$binDir"/gdal-config \
 		"$developDir"
 
+	if [ -n "$secondaryArchSuffix" -a -z "$commandSuffix" ]; then
+		rmdir "$binDir"
+	fi
 
 	if [ -n "$commandSuffix" ]; then
 		rm -rf "$binDir"
 	fi
```

Yeah, these two tests could be combined in a `if` `then` `else` `fi` with an `if` `then` `fi` inside one of its sides, but it would make things hard to understand.
Having two tests is nice because the first one is for removing the empty `bin/x86/`, and the second test is for dropping `binDir` when one builds gdal_x86 on x86_64 hybrid.

Reply:
Replacing the first "fi" with "else" should do it then?

Reply:
Nope. If you wanted to combine these two tests you would have to use something slightly different. This would work, for instance:
```bash
	if [ -z "$commandSuffix" ]; then
		if [ -n "$secondaryArchSuffix" ]; then
			rmdir "$binDir"
		fi
	else
		rm -rf "$binDir"
	fi
```

Reply:
But I think having two separate tests is better. When one reads these tests, one only takes 20 seconds to understand what case it covers. While for the combined tests one needs almost one minute to understand what the code does :)